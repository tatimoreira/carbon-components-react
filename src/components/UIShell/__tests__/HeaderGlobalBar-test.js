import React from 'react';
import { mount } from 'enzyme';
import { HeaderGlobalBar } from '../';

describe('HeaderGlobalBar', () => {
  it('should render', () => {
    const wrapper = mount(<HeaderGlobalBar className="custom-class" />);
    expect(wrapper).toMatchSnapshot();
  });
});
