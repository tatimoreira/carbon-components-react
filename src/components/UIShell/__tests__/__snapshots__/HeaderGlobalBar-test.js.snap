// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderGlobalBar should render 1`] = `
<HeaderGlobalBar
  className="custom-class"
>
  <div
    className="bx--platform-header__global custom-class"
  />
</HeaderGlobalBar>
`;
